packages/makeswift/src/utils/check-for-conflicting-files.ts

```diff
@@ -0,0 +1,8 @@
+import * as glob from 'glob'
+
+const CONFLICTING_FILES = ['pages/api/makeswift.*', 'pages/[...path].*', 'pages/_document.*']
```

Comment:
What about `[[...path]].*`?

Reply:
Does `'pages/api/makeswift.*'` match `pages/api/makeswift.ts` as well as `pages/api/makeswift/index.ts` or any file under a `pages/api/makeswift` directory?

Reply:
Would it be useful to write test for this?

Reply:
Just `pages/api/makeswift.ts` or `pages/api/makeswift.js` (because of the dot). Do we want to support the `pages/api/makeswift/index.ts` use case?

And I'm not sure this is the kind of thing that leans itself well to tests — I don't want to keep around some fake files just for that case

Reply:
And ah, thanks, the optional catch-all route should be on this list as well

Reply:
We _don't_ want them to have a folder named `pages/api/makeswift` or a file named `pages/api/makeswift.*` So I guess right now we're checking for the file, we want to also check for the folder

Reply:
Ok, adjusted for that

Reply:
(updated)